timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -260,6 +260,64 @@ limitations under the License.
       </div>
     </div>
 
+    <!-- write a v-dialog here that shows the shortcut help -->
+    <v-dialog v-model="showShortcutHelp" width="700" style="overflow: visible">
+      <v-card width="700" style="overflow: visible">
+        <v-card-title> Keyboard Shortcut help </v-card-title>
+
+        <v-card-text
+          ><v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Ctrl </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Shift </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> n </v-btn>
```

Comment:
Small optional nit: Since you are now using shift in the shortcut it would be fine to use uppercase letters again, since shift does make them uppercase.